ksp_plugin/flight_plan_optimizer.cpp

```diff
@@ -564,6 +694,90 @@ EvaluateGateauxDerivativeOfDistanceToCelestialWithReplacement(
   return (distance_δh - distance) / h;
 }
 
+Angle FlightPlanOptimizer::EvaluateRelativeInclinationlWithReplacement(
+    Celestial const& celestial,
+    NavigationFrame const& frame,
+    Angle const& target_inclination,
+    HomogeneousArgument const& homogeneous_argument,
+    NavigationManœuvre const& manœuvre,
+    int const index) {
+  auto const [time, barycentric_degrees_of_freedom] =
+      EvaluatePeriapsisWithReplacement(
+          celestial, homogeneous_argument, manœuvre, index);
+  auto const navigation_degrees_of_freedom =
+      frame.ToThisFrameAtTime(time)(barycentric_degrees_of_freedom);
+  auto const r = navigation_degrees_of_freedom.position() - Navigation::origin;
+  auto const v = navigation_degrees_of_freedom.velocity();
+  Angle const i =
+      AngleBetween(Wedge(r, v), Bivector<double, Navigation>({0, 0, 1}));
+  Angle const signed_relative_inclination = [](Angle const α) {
+    return α > π * Radian ? α - 2 * π * Radian : α;
+  }(Mod(i - target_inclination, 2 * π * Radian));
+  return signed_relative_inclination;
+}
+
+FlightPlanOptimizer::AngleGradient
+FlightPlanOptimizer::Evaluate𝛁RelativeInclinationWithReplacement(
+    Celestial const& celestial,
+    NavigationFrame const& frame,
+    Angle const& target_inclination,
+    HomogeneousArgument const& homogeneous_argument,
+    NavigationManœuvre const& manœuvre,
+    int const index) {
+  auto const angle =
+      EvaluateRelativeInclinationlWithReplacement(celestial,
+                                                  frame,
+                                                  target_inclination,
+                                                  homogeneous_argument,
+                                                  manœuvre,
+                                                  index);
+
+  AngleGradient gradient;
+  for (int i = 0; i < HomogeneousArgument::dimension; ++i) {
+    HomogeneousArgument homogeneous_argument_δi = homogeneous_argument;
```

Comment:
The use of `δi` in the context of inclination is rather confusing.  Should we pick, say, `n` for the index and use `δn`?

Reply:
k.